### PR TITLE
add read me for chromadb, popularity, lightfm and writer handlers

### PR DESCRIPTION
**Pull request by dusvyat**
## Description

add read me files for chromadb, writer, popularity and lightfm

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: mindsdb/integrations/handlers/chromadb_handler/Manual_QA.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Welcome to the MindsDB Manual QA Testing for Cassandra Handler
```
Comment (paxcema) [inline mindsdb/integrations/handlers/chromadb_handler/Manual_QA.md:1]: Rename all references to `Cassandra` with `ChromaDB` 😬

File: mindsdb/integrations/handlers/chromadb_handler/Manual_QA.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+> **Please submit your PR in the following format after the underline below `Results` section. Don't forget to add an underline after adding your changes i.e., at the end of your `Results` section.**
+
+## Testing Cassandra Handler with [Dataset Name](URL to the Dataset)
```
Comment (paxcema) [inline mindsdb/integrations/handlers/chromadb_handler/Manual_QA.md:5]: Also, remove this initial section with instructions.

File: mindsdb/integrations/handlers/chromadb_handler/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+This is the implementation of the ChromaDB for MindsDB.
+
+## ChromaDB
+
+Chroma is the open-source embedding database. Chroma makes it easy to build LLM apps by making knowledge, facts, and skills pluggable for LLMs.
+
+## Implementation
+
+This handler uses `chromadbd` python library connect to a chromadb instance, it uses langchain to make use of their pre-existing semantic search functionality
```
Comment (paxcema) [inline mindsdb/integrations/handlers/chromadb_handler/README.md:11]: typo: `chromadbd`

File: mindsdb/integrations/handlers/chromadb_handler/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+The required arguments to establish a connection are:
+
+* `chroma_api_impl`: the api implementation, likely should be `rest`
```
Comment (paxcema) [inline mindsdb/integrations/handlers/chromadb_handler/README.md:15]: Would recommend specifying what other choices exist for this param

File: mindsdb/integrations/handlers/chromadb_handler/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* `chroma_server_host`: the host name or IP address of the ChromaDB instance
+* `chroma_server_http_port`: the port to use when connecting
+* `embedding_function`: the embedding function to use to get embeddings from input text or search text
```
Comment (paxcema) [inline mindsdb/integrations/handlers/chromadb_handler/README.md:18]: Maybe worth pointing out that huggingface hub model strings work here

File: mindsdb/integrations/handlers/lightfm_handler/README.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+## Any directions for future work in subsequent versions of the handler?
+- Support for hybrid recommender by allowing users to pass in user and item features in addition to interaction data
+- Support for updating models with partial fit
+- Support for user-user recommendations
+
+
+## Please provide a minimal SQL example that uses this ML engine (pointers to integration tests in the PR also valid)
+```sql
+-- Create Writer engine
```
Comment (paxcema) [inline mindsdb/integrations/handlers/lightfm_handler/README.md:46]: typo: reference to `Writer`

**Review by paxcema — CHANGES_REQUESTED** [review]
Left a few observations on typos, but mostly LGTM!

**Comment by paxcema** [comment]
Friendly reminder to address the requested changes @dusvyat 👀

**Comment by dusvyat** [comment]
> Friendly reminder to address the requested changes @dusvyat 👀

@paxcema sorry completely forgot about this one, just fixed the typos you pointed out. Please feel free to +1 and merge when you get a sec

**Review by paxcema — APPROVED** [review]

**Pull request merged**